tests/driver_enc28j60/Makefile

```diff
@@ -0,0 +1,51 @@
+export APPLICATION = driver_enc28j60
+include ../Makefile.tests_common
+
+FEATURES_REQUIRED = periph_spi periph_gpio
+
+BOARD_INSUFFICIENT_MEMORY := msb-430h stm32f0discovery weio z1
+BOARD_BLACKLIST        := nucleo-f334
+# nucleo-f334:  not enough GPIO pins defined
+
+USEMODULE += gnrc_netdev2
+USEMODULE += gnrc_netif_default
+USEMODULE += auto_init_gnrc_netif
+USEMODULE += enc28j60
+USEMODULE += gnrc_ipv6_router_default
+USEMODULE += gnrc_icmpv6_echo
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+
+ifneq (,$(filter stm32f4discovery,$(BOARD)))
+  ENC_SPI = SPI_1
+  ENC_CS = GPIO_PIN\(PORT_B,12\)
+  ENC_INT = GPIO_PIN\(PORT_B,11\)
+  ENC_RST = GPIO_PIN\(PORT_B,10\)
+endif
+
+ifneq (,$(ENC_SPI))
```

Comment:
this could be more compact if you use ENC_SPI ?= SPI_0 and move this to line 19 (same for all following lines). A simple CFLAGS += -DENC_SPI=$(ENC_SPI) would then suffice instead of using the `ifneq .. else .. endif` construct